src/interp/interp.h

```diff
@@ -224,6 +229,11 @@ struct TypedValue {
           break;
       }
     }
+    // Nullable reftypes also have Nullref as a subtype
```

Comment:
is this just leftover diff that needs to be rebased?

Reply:
Yup